src/main/fc/parameter_names.h

```diff
@@ -197,3 +197,11 @@
 #define PARAM_NAME_GPS_LAP_TIMER_GATE_TOLERANCE "gps_lap_timer_gate_tolerance_m"
 #endif // USE_GPS_LAP_TIMER
 #endif
+
+#define PARAM_NAME_IMU_DCM_KP "imu_dcm_kp"
+#define PARAM_NAME_IMU_DCM_KI "imu_dcm_ki"
+#define PARAM_NAME_IMU_SMALL_ANGLE "imu_small_angle"
+#define PARAM_NAME_IMU_PROCESS_DENOM "imu_process_denom"
+#ifdef USE_MAG
+#define PARAM_NAME_IMU_MAG_DECLINATION "imu_mag_declination"
```

Comment:
Sorry to ask again, however while adjusting mag settings I would go 'get mag' in CLI, and this was the odd one out; it wasn't listed with the other `mag_...` parameters.  Although the code adjusting declination is done within the IMU part of our code base, I wonder if simply calling it `mag_declination` would be more transparent for the average user?

Reply:
Sure, good point. Should i just change it to
```
#define PARAM_NAME_IMU_MAG_DECLINATION "mag_declination"
```
so that in CLI it is listed with get mag, but in the codebase it is clear that it belongs to imu?

Reply:
maybe i should also change imu_small_angle back to small_angle? a lot of people will see this otherwise when porting.

Reply:
Ok, done.